rome-modules/src/test/java/com/rometools/modules/mediarss/MediaModuleTest.java

```diff
@@ -91,6 +89,30 @@ public void testParse() throws Exception {
         }
     }
 
+
+    public void testParseFileSizeWithUnits() throws Exception {
+        final SyndFeed feed = getSyndFeed("org/rometools/feed/module/mediarss/filesize-with-unit.xml");
+        final List<SyndEntry> entries = feed.getEntries();
+
+        final MediaEntryModule m1 = (MediaEntryModule) entries.get(0).getModule(MediaModule.URI);
+        assertThat(m1.getMediaContents()[0].getFileSize(), is(valueOf(2).multiply(valueOf(1024).pow(3)).longValue()));
+
+        final MediaEntryModule m2 = (MediaEntryModule) entries.get(1).getModule(MediaModule.URI);
+        assertThat(m2.getMediaContents()[0].getFileSize(), is(valueOf(1.1).multiply(valueOf(1024).pow(3)).longValue()));
+
+        final MediaEntryModule m3 = (MediaEntryModule) entries.get(2).getModule(MediaModule.URI);
+        assertThat(m3.getMediaContents()[0].getFileSize(), is(valueOf(2).multiply(valueOf(1024).pow(2)).longValue()));
+
+        final MediaEntryModule m4 = (MediaEntryModule) entries.get(3).getModule(MediaModule.URI);
+        assertThat(m4.getMediaContents()[0].getFileSize(), is(2L));
+
+        final MediaEntryModule m5 = (MediaEntryModule) entries.get(4).getModule(MediaModule.URI);
+        assertThat(m5.getMediaContents()[0].getFileSize(), is(valueOf(5.5).multiply(valueOf(1024).pow(4)).longValue()));
+
+        final MediaEntryModule m6 = (MediaEntryModule) entries.get(5).getModule(MediaModule.URI);
+        assertThat(m6.getMediaContents()[0].getFileSize(), is(valueOf(9000).multiply(valueOf(1024)).longValue()));
```

Comment:
Why 9000? Is this test about the number or the unit? Same for `5,5TB` and others.